Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -810,6 +810,24 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+    public override func visit(_ node: PatternBindingSyntax) -> SyntaxVisitorContinueKind {
+        if shouldSkip(node) {
+            return .skipChildren
+        }
+
+        if let unexpected = (node.pattern.as(TuplePatternSyntax.self)?.unexpectedBetweenElementsAndRightParen),
+            let previousToken = node.previousToken(viewMode: .sourceAccurate) {
```

Comment:
I think you should check that `unexpected` only consists of a single tokens, which is an identifier. Otherwise we also emit this diagnostic if we skipped over something completely unrelated. Same below.

Also, we should add the Fix-Its to join the identifiers together like we do in the other places where we emit this diagnostic.